packages/ui5-middleware-onelogin/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "ui5-middleware-onelogin",
+  "version": "1.0.1",
+  "description": "A universal login provider for UI5 tooling",
+  "main": "/lib/index.js",
+  "scripts": {
+    "test": "npx playwright test"
+  },
+  "keywords": [
+    "UI5-tooling",
+    "SAPUI5",
+    "OPENUI5",
+    "Fiori"
+  ],
+  "repository": {
+    "url": "https://github.com/uxkjaer/ui5-middleware-onelogin.git"
+  },
+  "author": "Jakob Kjaer",
+  "license": "MIT",
+  "dependencies": {
+    "@ui5/logger": "^2.0.1",
+    "async-prompt": "*",
+    "cookie": "^0.4.1",
+    "dotenv": "^12.0.3",
+    "playwright-chromium": "^1.18.0",
+    "playwright-core": "^1.18.0",
+    "sleep-promise": "^9.1.0"
+  },
+  "devDependencies": {
+    "@playwright/test": "^1.18.0",
+    "@types/cookie": "^0.4.1",
+    "@types/express": "^4.17.13",
+    "@types/prompt": "^1.1.2",
+    "eslint": "^8.7.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "husky": "^7.0.4",
+    "lint-staged": "^12.2.0",
+    "prettier": "2.5.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
I don't know if the husky hook works for monorepos in the nested packages. This should most probably be handled in the workspace root and then call the packages. I think we could use Lerna to run lint-staged for the pre-commit hook? This is BTW a general open topic for the repo...

Reply:
That was from when it was in it's own library. Removed